packages/datetime/src/dateInput.tsx

```diff
@@ -68,6 +70,13 @@ export interface IDateInputProps extends IDatePickerBaseProps, IProps {
     format?: string;
 
     /**
+     * Props to pass to the input group.
+     * `disabled` and `value` will be ignored in favor of the top-level props on this component.
+     * Note that `ref` will also be ignored; use `inputRef` instead.
```

Comment:
Is this ref note also true of how `DateRangeInput` uses `{start,end}InputProps`? We should add a note there if so, maybe even in this PR.

Also, worth mentioning that `type` is ignored?